helm_charts_updater/models.py

```
@@ -1,50 +1,120 @@
+    condition: str | None = None
+    tags: list[str] | None = None
+    importValues: list[str] | None = None
+    alias: str | None = None
 
     @field_validator("version", mode="before")
-    def validate_version(cls, v) -> Optional[str]:
-        if not semver.VersionInfo.is_valid(v):
-            raise ValueError(f"{v} is not a valid version")
-        return v
+    @classmethod
+    def validate_version(cls, v: Any) -> str:
```

Ruff (0.14.13)

Dynamically typed expressions (typing.Any) are disallowed in `v`

(ANN401)